daemon/daemon_windows.go

```diff
@@ -340,7 +340,9 @@ func (daemon *Daemon) initNetworkController(config *Config, activeSandboxes map[
 
 		// If there is no nat network create one from the first NAT network
 		// encountered
-		if !defaultNetworkExists && runconfig.DefaultDaemonNetworkMode() == containertypes.NetworkMode(strings.ToLower(v.Type)) {
+		if !defaultNetworkExists &&
+			runconfig.DefaultDaemonNetworkMode() == containertypes.NetworkMode(strings.ToLower(v.Type)) &&
+			n == nil {
```

Comment:
This change looks unrelated. Could you update the comment (or is that still accurate?)